keras_contrib/backend/cntk_backend.py

```diff
@@ -1,2 +1,10 @@
 from keras.backend import cntk_backend as KCN
 import cntk as C
+import numpy as np
+
+
+def clip(x, min_value, max_value):
+    if max_value is None:
```

Comment:
if max_value defaults to inf, should min default to negative infinity?

Reply:
You're correct. The Keras CNTK backend uses -inf as the default min_value. I did it this way because the TensorFlow backend *doesn't* default to -inf for min_value ([ref](https://github.com/fchollet/keras/blob/2.0.8/keras/backend/tensorflow_backend.py#L1472-L1489)), but failed to notice that the behavior of the CNTK backend differed.

Reply:
I changed my mind and extended all backends to allow `None` for `min_value`. See notes in `tests/keras_contrib/backend/backend_test.py` for rationale and details.

Reply:
Documentation needs to be updated accordingly. Could these changes be valuable in keras itself?